db/primary_migrate/20241129190947_add_index_to_profiles_on_active_and_idv_level.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToProfilesOnActiveAndIdvLevel < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :profiles, %i[active idv_level verified_at], algorithm: :concurrently, name: 'index_profiles_on_active_and_idv_level_and_verified_at'
+  end
+end
```

Comment:
This shouldn't have too many writes. It should significantly speed up the query above.

Reply:
Does the report run fast enough without the index?

Reply:
I think in general we should avoid creating indexes if possible. The current query that would use this runs quickly enough, and we'll hopefully be able to avoid using Postgres for it soon.